src/s3.rs

```diff
@@ -11784,9 +11796,15 @@ impl<P, D> S3Client<P, D> where P: ProvideAwsCredentials, D: DispatchSignedReque
     pub fn create_bucket(&self, input: &CreateBucketRequest) -> Result<CreateBucketOutput, S3Error> {
         let region = Region::UsEast1;
         let mut create_config : Vec<u8>;
-        let mut request = SignedRequest::new("PUT", "s3", region, "");
-        let hostname = self.hostname(Some(&input.bucket));
-        request.set_hostname(Some(hostname));
+        let mut path = String::from("/");
```

Comment:
In the case where the input bucket is dns compatible, this results in a path of "/", whereas the original had a path of "".  My guess is that's fine (I'll run the integration tests to confirm).  Just making a note as I read through the code.